spec/models/validations/household_validations_spec.rb

```diff
@@ -637,4 +637,20 @@
       end
     end
   end
+
+  describe "housing needs validations" do
+    it "is invalid when housingneeds == 1 && housingneeds_type == 3 && housingneeds_other == 0" do
+      record.housingneeds = 1
+      record.housingneeds_type = 3
+      record.housingneeds_other = 0
+
+      household_validator.validate_housing_needs(record)
+
+      error_message = ["If somebody in the household has disabled access needs, they must have the access needs listed, or other access needs"]
+
+      expect(record.errors["housingneeds"]).to eq(error_message)
+      expect(record.errors["housingneeds_type"]).to eq(error_message)
+      expect(record.errors["housingneeds_other"]).to eq(error_message)
+    end
+  end
```

Comment:
can the name of the test be written in english not obscure codes, I don't know off the top of my head what housingneeds == 1 means in terms of what the user answered
and slight pickiness error_message should be a string and the expects can be `to include error_message`